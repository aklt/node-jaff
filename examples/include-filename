#begin coffee
remove = __dirname.length
#end
filename: {__filename}
#include include/filename
TODO: add __dirname when contexts are fixed
