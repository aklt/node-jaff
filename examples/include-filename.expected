filename: include-filename
filename: filename
TODO: add __dirname when contexts are fixed
